packages/mobile/src/identity/verification.ts

```diff
@@ -211,13 +222,23 @@ export function* doVerificationFlow() {
     return true
   } catch (error) {
     Logger.error(TAG, 'Error occured during verification flow', error)
+    error.message === ErrorMessages.INSUFFICIENT_BALANCE
+      ? yield put(setVerificationStatus(VerificationStatus.InsufficientBalance))
+      : yield put(setVerificationStatus(VerificationStatus.Failed))
+
     yield put(showErrorOrFallback(error, ErrorMessages.VERIFICATION_FAILURE))
-    yield put(setVerificationStatus(VerificationStatus.Failed))
     return error.message
   }
 }
 
-// Requests if necessary additional attestations and returns all revealable attetations
+export function* balanceSufficientForAttestations(attestationsRemaining: number) {
+  const userBalance = yield select(stableTokenBalanceSelector)
```

Comment:
it doesn't matter here, but if you wanted to make this function sync so you could use it in the UI, you can always do `stableTokenBalanceSelector(store.getState())`